### PR TITLE
Added new options to the settings dashboard tab.

### PR DESCRIPTION
**Pull request by diabolical17**

### CODE REVIEW

File: pages/dashboard/misc/revoke-token.vue [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    </div>
+    <section class="essentials pad-maker">
+      <p>
+        Revoking your Modrinth token can have unintended consequences. Please be
+        wary, the following could break:
+      </p>
+      <ul>
+        <li>Any application that uses your token to access the API.</li>
+        <li>
+          Gradle - If Minotaur is given a incorrect token, your gradle builds
```
Comment (falseresync) [inline pages/dashboard/misc/revoke-token.vue:14]: Lowercase the `If`

File: pages/dashboard/misc/revoke-token.vue [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          could fail.
+        </li>
+        <li>
+          GitHub - If you use a GitHub action that uses the Modrinth API, it
```
Comment (falseresync) [inline pages/dashboard/misc/revoke-token.vue:18]: Lowercase the `If`

File: pages/dashboard/misc/revoke-token.vue [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+      </ul>
+      <p>If you are willing to continue, complete the following steps:</p>
+      <ol>
+        <li>
+          <a
+            target="_blank"
+            href="https://github.com/settings/connections/applications/3acffb2e808d16d4b226"
+            >Head to the Modrinth Application page on GitHub.</a
+          >
+          Make sure to be logged into the GitHub account you used for modrinth!
```
Comment (falseresync) [inline pages/dashboard/misc/revoke-token.vue:30]: Capitalize `modrinth`

**Review by falseresync — CHANGES_REQUESTED** [review]
Other than this, looks good

**Comment by diabolical17** [comment]
Should be all done and dandy

**Review by falseresync — APPROVED** [review]

**Pull request merged**